tests/DecisionServiceTests/DecisionServiceTest.php

```diff
@@ -1133,4 +1133,54 @@ public function testGetVariationForFeatureRolloutWhenUserDoesNotQualifyForAnyTar
 
         $this->assertNull($this->decisionService->getVariationForFeatureRollout($feature_flag, 'user_1', $user_attributes));
     }
+
+    public function testGetVariationForFeatureRolloutWhenUserDoesNotQualifyForAnyTargetingRulesAndUserIsNotBucketedInTheEveryoneElseRule()
+    {
```

Comment:
I don't think we need to write a new test for this. There is a test written already. We can add our log message assertion in that,